<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountTab</class>
 <widget class="QWidget" name="AccountTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="logonNameLabel">
       <property name="text">
        <string>Logon name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="upn_prefix_edit"/>
       </item>
       <item>
        <widget class="QComboBox" name="upn_suffix_edit"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="unlockAccountLabel">
       <property name="text">
        <string>Unlock account:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="unlock_check"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="accountExpiresLabel">
       <property name="text">
        <string>Account expires:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QFrame" name="frame">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>11</number>
        </property>
        <property name="topMargin">
         <number>11</number>
        </property>
        <property name="rightMargin">
         <number>11</number>
        </property>
        <property name="bottomMargin">
         <number>11</number>
        </property>
        <item>
         <widget class="ExpiryWidget" name="expiry_widget" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="logon_hours_button">
       <property name="text">
        <string>Logon hours...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="logon_computers_button">
       <property name="text">
        <string>Logon computers...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Account options:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>449</width>
        <height>276</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="disabled_check">
         <property name="text">
          <string>Account disabled</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cant_change_pass_check">
         <property name="text">
          <string>User cannot change password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="pass_expired_check">
         <property name="text">
          <string>User must change password on next logon</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="dont_expire_pass_check">
         <property name="text">
          <string>Don't expire password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="reversible_encrypt_check">
         <property name="text">
          <string>Store password using reversible encryption</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="smartcard_check">
         <property name="text">
          <string>Smartcard is required for interactive logon</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cant_delegate_check">
         <property name="text">
          <string>Account is sensitive and cannot be delegated</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="des_key_check">
         <property name="text">
          <string>Use Kerberos DES encryption types for this account</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="require_preauth_check">
         <property name="text">
          <string>Don't require Kerberos pre-authentication</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Password settings:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pso_label">
       <property name="text">
        <string>PLACEHOLDER</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExpiryWidget</class>
   <extends>QWidget</extends>
   <header>attribute_edits/expiry_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
